src/ngraph/op/non_zero.cpp

```diff
@@ -22,14 +22,23 @@ using namespace std;
 
 constexpr NodeTypeInfo op::v3::NonZero::type_info;
 
-op::v3::NonZero::NonZero(const Output<Node>& arg)
+op::v3::NonZero::NonZero(const Output<Node>& arg, const std::string& index_element_type)
```

Comment:
Spec says, output_type is optional with default i64 value -- we can avoid massive code changes.

Reply:
Done